packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1356,4 +1356,14 @@
 	<symbol id="kv-support" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
 		<path d="M12 22a9.738 9.738 0 0 1-3.9-.788 10.099 10.099 0 0 1-3.175-2.137c-.9-.9-1.612-1.958-2.137-3.175A9.738 9.738 0 0 1 2 12c0-1.383.263-2.683.788-3.9a10.099 10.099 0 0 1 2.137-3.175c.9-.9 1.958-1.612 3.175-2.137A9.738 9.738 0 0 1 12 2c1.383 0 2.683.263 3.9.788a10.098 10.098 0 0 1 3.175 2.137c.9.9 1.613 1.958 2.137 3.175A9.738 9.738 0 0 1 22 12a9.738 9.738 0 0 1-.788 3.9 10.098 10.098 0 0 1-2.137 3.175c-.9.9-1.958 1.613-3.175 2.137A9.738 9.738 0 0 1 12 22Zm-2.9-2.55 1.2-2.75a4.677 4.677 0 0 1-1.813-1.162A5.672 5.672 0 0 1 7.3 13.7l-2.75 1.15c.383 1.067.975 2 1.775 2.8.8.8 1.725 1.4 2.775 1.8ZM7.3 10.3c.283-.7.68-1.313 1.188-1.838A4.677 4.677 0 0 1 10.3 7.3L9.15 4.55c-1.067.4-2 1-2.8 1.8-.8.8-1.4 1.733-1.8 2.8L7.3 10.3ZM12 15c.833 0 1.542-.292 2.125-.875A2.893 2.893 0 0 0 15 12c0-.833-.292-1.542-.875-2.125A2.893 2.893 0 0 0 12 9c-.833 0-1.542.292-2.125.875A2.893 2.893 0 0 0 9 12c0 .833.292 1.542.875 2.125A2.893 2.893 0 0 0 12 15Zm2.9 4.45a7.78 7.78 0 0 0 2.763-1.787A7.78 7.78 0 0 0 19.45 14.9l-2.75-1.2c-.25.7-.633 1.304-1.15 1.813a5.597 5.597 0 0 1-1.8 1.187l1.15 2.75Zm1.8-9.2 2.75-1.15a7.78 7.78 0 0 0-1.787-2.762A7.78 7.78 0 0 0 14.9 4.55l-1.15 2.8c.683.25 1.275.63 1.775 1.138.5.508.892 1.095 1.175 1.762Z"/>
 	</symbol>
+	
+	<symbol id="kv-dotted-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
This view box must be 24x24. Please update this on both svgs.

Reply:
not only update this property. It also necessary to resize the SVG to match that viewbox size. If we don't do that we can have a box with 24px but the icon doesn't fit all the box.

Reply:
I've replaced the icon with the 24x24 version, optimized :)